lib/training-module.js

```diff
@@ -1,5 +1,14 @@
+const { sample } = require('lodash');
 const EntityMigrator = require('./entity-migrator');
 
+const species = [
+  'Rats and mice',
+  'Ferrets',
+  'Fish',
+  'Non-human primates',
+  'Equidae'
+];
```

Comment:
Is `species` a required field? My guess is that it isn't, and so we should include a `null` option here.